src/scss/components/typography/_index.scss

```diff
@@ -64,9 +64,9 @@ hx-error {
 // Additional Helper Classes
 .hxSubdued {
   color: $gray-700;
-  font-size: 0.875rem;
+  font-size: 0.625rem;
   font-weight: 400;
-  line-height: 1.5rem;
+  line-height: 1;
```

Comment:
We should make these changes specific to the Pill component.

Reply:
Changes have been done, Please review